requirements/integration_test_requirements.txt

```diff
@@ -1,6 +1,6 @@
 # For random forest model
 scikit-learn
 # For BNN model
--e git+https://github.com/automl/pybnn.git#egg=pybnn
-# For Lating design
+pybnn==0.0.3
```

Comment:
could you update that to pybnn==0.0.4? That should fix the unittests

Reply:
Done

Reply:
I am sorry Mark, I've just found another small bug. Could you set it to pybnn==0.0.5? That fixed at least my local integration tests

Reply:
Will do!